### DISCUSSION TITLE
Bulk delete drafts

### DISCUSSION THREAD

**Discussion by laurabedoya1** [GitbookIO/community#303] [Feature Requests]
Option to delete drafts in bulk, please. Manually deleting is very time consuming if you have a lot of drafts.

**Comment by meretciel** [comment]
Hi laurabedoya1, I'm wondering how you manage to manually delete a single draft? I don't find any thing related to delete. I can change the type though (draft - change request - archive) and I'm unable to delete any of them. Thanks in advance!

**Reply by laurabedoya1** [reply]
I had to delete 70+ drafts manually. I'm not sure why you can't delete any of them; it might be a separate issue!

**Reply by meretciel** [reply]
Thanks for the reply and happy new year!
